File: Examples/CSharp/KnowledgeBase/ComparingVSTOWithAspose/UsingAsposeCells.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Cells.Examples.CSharp.KnowledgeBase.ComparingVSTOWithAspose
{
    public class UsingAsposeCells
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            DateTime start = DateTime.Now;
            string myPath = dataDir + @"TempBook.xls";
            Workbook workbook = new Workbook(myPath);
            Worksheet ws = workbook.Worksheets[0];

            for (int i = 0; i < 1000; i++)
            {
                for (int j = 0; j < 20; j++)
                {
                    ws.Cells[i, j].PutValue("Row " + (i + 1).ToString() + " " + "Col " + (j + 1).ToString());
                }
            }

            workbook.Save(dataDir + @"TempBook1.xls");
            DateTime end = DateTime.Now;
            TimeSpan time = end - start;
            Console.WriteLine("File Created! " + "Time consumed (Seconds): " + time.TotalSeconds.ToString());
            // ExEnd:1
        }
    }
}

File: Examples/VisualBasic/KnowledgeBase/ComparingVSTOWithAspose/UsingAsposeCells.vb
Namespace KnowledgeBase.ComparingVSTOWithAspose
    Public Class UsingAsposeCells
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim start As DateTime = DateTime.Now
            Dim myPath As String = dataDir & Convert.ToString("TempBook.xls")
            Dim workbook As New Workbook(myPath)
            Dim ws As Worksheet = workbook.Worksheets(0)

            For i As Integer = 0 To 999
                For j As Integer = 0 To 19
                    ws.Cells(i, j).PutValue("Row " + (i + 1).ToString() + " " + "Col " + (j + 1).ToString())
                Next
            Next

            workbook.Save(dataDir & Convert.ToString("TempBook1_out.xls"))
            Dim [end] As DateTime = DateTime.Now
            Dim time As TimeSpan = [end] - start
            Console.WriteLine("File Created! " + "Time consumed (Seconds): " + time.TotalSeconds.ToString())
            ' ExEnd:1
        End Sub
    End Class
End Namespace